modules/eco-node-health-check-operator-about.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/node-health-check-operator-installation.adoc
+
+[id="about-node-health-check-operator_{context}"]
+= About the Node Health Check Operator
+
+The Node Health Check Operator deploys the `NodeHealthCheck` controller, which in turn creates the `NodeHealthCheck` custom resource (CR). The Node Health Check Operator also installs the Poison Pill Operator as a default remediation provider.
+
+The Operator uses the controller to detect the health of a node in the cluster. When the controller detects an unhealthy node, it creates a `NodeHealthCheck` custom resource (CR), which defines a set of criteria and thresholds to determine the node's health.
```

Comment:
The operator manages the NodeHealthCheck CRs. It also creates a default on when the operator starts, if there isn't one already.

When the node detects an unhealthy node it creates another CR, of another type, for the remediation. The details of that CR, the Group Version and Kind are set in the NodeHealthCheck CR (which is what line 12 says)

Reply:
The reason for the comment above is because this sentence 
"When the controller detects an unhealthy node, it creates a `NodeHealthCheck` custom resource (CR), which defines a set of criteria and thresholds to determine the node's health."

First the NodeHealthCheck CR needs to exist. there it has the set of conditions to check. See `unhealthyConditions` in the README.md - https://github.com/medik8s/node-healthcheck-operator#nodehealthcheck-custom-resource

When the operator decides the node is unhealthy it creates another resource based on the `remediationTemplate`